src/Uno.Templates/content/unoapp/.template.config/TemplateWizard.json

```diff
@@ -126,7 +128,7 @@
 					"mauiEmbedding": "false",
 					"wasmMultiThreading": "false",
 					"dspGenerator": "true",
-					"themeService": "true",
+					"themeService": "false",
```

Comment:
This should default to true in the wizard experience for recommended, which is why it should be true

Reply:
This was always the default in Recommended, the TemplateWizard went rogue.